tasks.py

```diff
@@ -32,10 +32,12 @@
 default_network = "kind"
 extra_network = "network2"
 controller_gen_version = "v0.14.0"
+kubectl_version = "v1.27.0"
+kustomize_version = "v5.4.0"
 build_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "build")
 kubectl_path = os.path.join(build_path, "kubectl")
 controller_gen_path = os.path.join(build_path, "bin", "controller-gen")
-kubectl_version = "v1.27.0"
+kustomize_path = os.path.join(build_path, "bin", "kustomize")
```

Comment:
can you also use this in place where we currently call kustomize straight? IIUC should only be here https://github.com/metallb/metallb/blob/28b13e36bafc10f03557b76be83f6a8941136566/tasks.py#L324 (also, call fetch_kustomize on top of the function)

Reply:
Hi, I've added the fetch_kustomize on top of the function as requested, 
For what concerns the comment
> can you also use this in place where we currently call kustomize straight
I may be misunderstanding, but as it is now I should be already using that `kustomize_path`  when we call kustomize. Did you find any place where I'm missing it?

Reply:
ah, apologies. I did a quick search and found https://github.com/metallb/metallb/blob/28b13e36bafc10f03557b76be83f6a8941136566/tasks.py#L335 where we use kubectl kustomize. Ignore my comment

Reply:
Actually I'm using it in my `fetch_kustomize` function

Reply:
yep but the point of `fetch_kustomize` is to fetch it to use it :-)
If we use `kubectl kustomize` there is no point in even fetching it

Reply:
so, let's use kustomize in https://github.com/metallb/metallb/blob/28b13e36bafc10f03557b76be83f6a8941136566/tasks.py#L335 
instead of kubectl kustomize.

Does it make sense?